<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBotDialog</class>
 <widget class="QDialog" name="AddBotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add bot</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e-mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Demonbuddy.exe path</string>
   </property>
   <widget class="QRadioButton" name="defaultDBPathRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>default</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="customDBPathRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>custom:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="customDBPathLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectDBPathButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="dbKeyLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Demonbuddy key</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="noflashCheckBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>129</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Don't flash D3 window</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="autostartCheckBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>132</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Start bot automatically</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="noupdateCheckBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>124</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Disable update check</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>34</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="profileLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectProfileButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>dbKeyLineEdit</tabstop>
  <tabstop>profileLineEdit</tabstop>
  <tabstop>selectProfileButton</tabstop>
  <tabstop>customDBPathLineEdit</tabstop>
  <tabstop>selectDBPathButton</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>noflashCheckBox</tabstop>
  <tabstop>autostartCheckBox</tabstop>
  <tabstop>noupdateCheckBox</tabstop>
  <tabstop>defaultDBPathRadioButton</tabstop>
  <tabstop>customDBPathRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddBotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddBotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
